packages/gatsby-telemetry/src/send.ts

```diff
@@ -1,10 +1 @@
-import { AnalyticsTracker } from "./telemetry"
-const instance = new AnalyticsTracker()
-
-function flush(): void {
-  instance.sendEvents().catch(_e => {
-    // ignore
-  })
-}
-
-flush()
+// no-op
```

Comment:
this file can be deleted, I think?

Reply:
Very possible, yes. 

The tricky bit here is that `gatsby-telemetry` doesn't have well defined public API surface and some other packages use non-main modules (and there is no export map) and I just am not sure that this module was never used by other packages and decided to keep module there. This module was not used by latest versions of packages (that part I did check) but I don't know about all the previous versions within current major.

The main goal of those changes was ensuring this new version will just not POST to now disabled collection endpoint - there is still quite a bit of code that is left in this package so I think this is reasonable to keep that module in